frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/cover/change_cover_popover.dart

```diff
@@ -231,13 +259,34 @@ class _ChangeCoverPopoverState extends State<ChangeCoverPopover> {
                       borderRadius: Corners.s8Border,
                     ),
                   ),
-                );
-              },
+                ),
+                Positioned(
```

Comment:
I think it is more appropriate for hover to display the delete button on the image.